AWS/connectors/trigger_sqs_messages.py

```diff
@@ -0,0 +1,68 @@
+"""Contains AwsSqsMessagesTrigger."""
+from functools import cached_property
+
+import orjson
+from sekoia_automation.connector import DefaultConnectorConfiguration
+
+from aws_helpers.sqs_wrapper import SqsConfiguration, SqsWrapper
+from connectors import AbstractAwsConnector
+
+
+class AwsSqsMessagesTriggerConfiguration(DefaultConnectorConfiguration):
+    """Contains configuration for AwsSqsMessagesTrigger."""
+
+    frequency: int = 60
```

Comment:
This parameter seems to not be used in the connector? Is useless or a `asyncio.sleep` is missing?

Reply:
Yep! Thank you! Fixed